app/controllers/CacheController.scala

```diff
@@ -145,14 +150,16 @@ class CacheController @Inject() (
     }
 
   private def getUserAnswers[T](lrn: String)(f: UserAnswers => T)(implicit writes: Writes[T]): Action[AnyContent] =
-    authenticate().async {
+    (authenticate() andThen getVersion).async {
       implicit request =>
         val eoriNumber = request.eoriNumber
         cacheRepository
           .get(lrn, eoriNumber)
           .map {
-            case Some(userAnswers) =>
+            case Some(userAnswers) if isRequestTransitional() == userAnswers.isTransitional =>
```

Comment:
Very minor but think we can get rid of the isRequestTransitional method now and just make this `case Some(userAnswers) if request.phase.isTransitional == userAnswers.isTransitional`